portfolio/src/main/webapp/script.js

```diff
@@ -44,10 +44,14 @@ function getServerMessage() {
     }); 
 }
 
-
 /** Creates an <li> element containing text. */
 function createListComments(text){
     const liElement = document.createElement('li');
-    liElement.innerText = text;
+    liElement.className = 'text';
+
+    const titleElement = document.createElement('span');
```

Comment:
same as above. I'm not sure why you're wrapping this in a span if there's no styling change.

Reply:
Noted, thanks